backend/src/modules/Boards/controller/BoardsController.js

```diff
@@ -27,5 +27,16 @@ export class BoardsController {
           response.status(400).send({ message: error.message });
         });
     });
+    this.router.route('/announcements/:id').delete((request, response) => {
+      const boardId = request.params.id;
+      this.service
+        .deleteAnnouncement(boardId)
+        .then((returnedData) => {
+          response.status(200).send({ returnedData });
+        })
+        .catch((error) => {
+          response.status(400).send({ message: error.message });
+        });
+    });
```

Comment:
Skoro ja mam taką samą ścieżkę to czy to zadziała?

Reply:
To jest niepotrzebne, zostało mi po testach z postmanem. Wyrzucam